CONTRIBUTING.md

```diff
@@ -72,58 +72,52 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
 To avoid conflicts and make the process faster, the FCP may be ended early when a new Minecraft version is released and
 QM is updated.
 
-### `T: new mappings`
+### `T: new`
 
 **Description**: Used for pull requests that focus mainly on adding new mappings.
 
 **Required Approvals**: 2
-- At least 1 approval must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 1 approval must come directly from a Mappings team member.
 
 **Final Comment Period**:
-- Snapshots: 1 day
+- Snapshots: 2 days (1 day with 3+ approvals)
 - Releases: 2 days
 
 **Special cases**:
-- If the PR adds new mappings needed by QSL, the Final Comment Period is shortened to 12 hours.
+- If the PR adds new mappings needed by QSL, the Final Comment Period is shortened to 12 hours, and it can be skipped if it receives 3 approvals or more.
 
 ### `T: refactor`
 
 **Description**: Used for pull request that focus mainly on changing the names of existing mappings.
 
 **Required Approvals**: 3
-- At least 2 approvals must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 2 approvals must come directly from a Mappings team member.
 
-**Final Comment Period**:
-- Snapshots: 3 days
-- Releases: 6 days
+**Final Comment Period**: 2 days
 
 ### `T: documentation`
 
 **Description**: Used for pull requests that focus mainly on adding or changing documentation.
 
 **Required Approvals**: 2
-- At least 1 approval must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 1 approval must come directly from a Mappings team member.
```

Comment:
I don't like having a single point here, I think we have a big enough team that we can still say 2 reviews are necessary

Reply:
ah I misinpreted that as eliminating the requirement for a triage review